pangeo_forge_recipes/recipes/base.py

```diff
@@ -50,6 +52,14 @@ def to_beam(self):
     def sha256(self):
         return dataclass_sha256(self, ignore_keys=self._hash_exclude_)
 
+    def get_execution_context(self):
+        return dict(
+            # See https://stackoverflow.com/a/2073599 re: version
+            version=pkg_resources.require("registrar")[0].version,
```

Comment:
Why track the registrar version? Ideally the target contents don't depend on the registrar. I was thinking we could track the pangeo_forge_recipes package version.

Reply:
Good catch! I copy-and-pasted this from the registrar, and forgot to update that.

Also this comment reminds me to add this PR to my TODO list of the week. 😄

Reply:
fixed in https://github.com/pangeo-forge/pangeo-forge-recipes/pull/359/commits/3ca6066b4f9ad6b19747f70a0c5ae11b779589da

Reply:
[]()